automation/deploy_pgcluster.yml

```diff
@@ -83,6 +83,15 @@
         - pgbackrest_auto_conf | default(true) | bool  # to be able to disable auto backup settings
       tags: always
 
+    - name: Make sure that the python3-cryptography package is present
+      ansible.builtin.package:
+        name: python3-cryptography
+        state: present
+      register: pack_status
+      until: pack_status is success
+      delay: 5
+      retries: 3
```

Comment:
Perhaps this should be part of the `tls_secrificate/generate` role.

Reply:
Agree.